pastas/model.py

```diff
@@ -889,7 +892,9 @@ def _get_sim_index(self, tmin, tmax, freq, warmup, update_sim_index=False):
                 break
 
         if self.sim_index is None or update_sim_index:
+            # TODO: sort out what to do for freq > "D"
```

Comment:
Same comment as above here, modeling on non-daily timestep can get tricky. Needs a look for 1.0